src/SamplesApp/SamplesApp.UnitTests.Shared/Controls/UITests/Presentation/SampleChooserViewModel.cs

```diff
@@ -756,7 +779,10 @@ public async Task<object> UpdateContent(CancellationToken ct, SampleChooserConte
 
 			if (newContent.ViewModelType != null)
 			{
+				Console.WriteLine("UpdateContent Creating VM");
```

Comment:
Codacy found an issue: [Remove this logging statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6226524)